.buildkite/hooks/post-command

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+set -euo pipefail
+[[ -n "${DEBUG:-}" ]] && set -x
+
+# Only run purge script on Windows agents.
+if [[ ${BUILDKITE_AGENT_META_DATA_OS} == "windows" ]]; then
```

Comment:
It looks like `BUILDKITE_AGENT_META_DATA_OS` isn't always defined on `trigger-pipelines`, which causes this to fail sometimes.  `BUILDKITE_AGENT_META_DATA_PLATFORM` contains the same information and is always present.

```suggestion
if [[ ${BUILDKITE_AGENT_META_DATA_PLATFORM} == "windows" ]]; then
```